.drone.yml

```diff
@@ -9,12 +9,20 @@ platform:
 
 steps:
   - name: mavenjdk8
-    pull: always
-    image: docker.io/maven:3.6.3-jdk-8-slim
+    pull: if-not-exists
+    image: adoptopenjdk/openjdk8-openj9:ubi
```

Comment:
Please use an Adoptium Image

Reply:
OpenJ9 is available on adoptium. I think every line of code was commented, so let's rather focus on your PR instead.